tests/data/test_converter_public_trades.py

```diff
@@ -0,0 +1,288 @@
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from freqtrade.data.converter import populate_dataframe_with_trades, public_trades_to_dataframe
+from freqtrade.data.converter.converter import trades_to_volumeprofile_with_total_delta_bid_ask
+
+
+BIN_SIZE_SCALE = 0.5
+
+
+def read_csv(filename, converter_columns: list = ['side', 'type']):
+    return pd.read_csv(filename, skipinitialspace=True, infer_datetime_format=True, index_col=0,
+                       parse_dates=True, converters={col: str.strip for col in converter_columns})
+
+
+@pytest.fixture(scope="module")
+def populate_dataframe_with_trades_dataframe():
+    return pd.read_feather('tests/testdata/populate_dataframe_with_trades_DF.feather')
```

Comment:
there's a fixture - `testdatadir` - which provides safe access to the testdata directory.

you're currently taking the assumption that the tests are executed from the root - which is not something that's always guaranteed.

(obviously applies to all below fixtures)

I'd also move all of the files needed for "this" test-module into a subdirectory - so they're clearly identifiable as "belongs to this functionality" ...